### ISSUE TITLE
Edit / Delete synchronization credentials

### ISSUE THREAD

**Issue by TheFreeman** [PhilippC/keepass2android#698]
Hi,

it would be so nice, if there is better usability on synchronizing my databases!
There´s no possibility to edit current connection credentials for the sync.
No possibility to find a list of entered connections, to delete them or to edit them.
Very frustrating, if your current sync uses the wrong/old password.
Entering new data is not working. It uses always the old one!

Thank you in advance.
kind regards!

**Comment by PhilippC** [comment]
I agree that all this could be better. But did you notice the three vertical dots in the "recent files list". If you click them, you can edit connection settings!
